commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -334,6 +346,13 @@ public void validateArgs() {
     final PicoCliTlsServerOptionsValidator picoCliTlsServerOptionsValidator =
         new PicoCliTlsServerOptionsValidator(spec, picoCliTlsServerOptions);
     picoCliTlsServerOptionsValidator.validate();
+
+    if (isMetricsEnabled() && metricsPushOptions.isMetricsPushEnabled()) {
```

Comment:
same as above, might be a good idea to either check for null of call getMetricsPushOptions method.